src/resource_clients/request_queue.ts

```diff
@@ -226,9 +231,9 @@ export class RequestQueueClient extends ResourceClient {
         const operationsInProgress = [];
         const individualResults = [];
 
-        // Send up to `maxParallelRequests` requests at once, wait for all of them to finish and repeat
-        for (let i = 0; i < requests.length; i += 25) {
-            const requestsInBatch = requests.slice(i, i + 25);
+        // Send up to `maxParallel` requests at once, wait for all of them to finish and repeat
+        for (let i = 0; i < requests.length; i += MAX_REQUESTS_PER_BATCH_OPERATION) {
+            const requestsInBatch = requests.slice(i, i + MAX_REQUESTS_PER_BATCH_OPERATION);
```

Comment:
We could simplify this using `async.mapLimit` and `_.chunk`, the code would be a lot cleaner, and the operations would run with the specified concurrency all the time, instead of just having batches that are processed in parallel.

Reply:
We would need to add a dependency on `async`, right? I don't really want to do that.

Yeah as said in slack, I'm working on the concurrency, I will add it to this pr

Reply:
We're already using https://github.com/vercel/async-retry for the regular exponential backoff so you can use this if it helps.